src/main/resources/application.yml

```diff
@@ -13,6 +13,9 @@ application:
       some_val: "Hello World"
     query:
       another_val: 123
+  httpPost:
+    headers:
+      Content-Type: "application/json"
```

Comment:
How is it solved with other request types? GET in the first place but other ones in the future as well.

If it's something that needs finding answers from the code in the future, add a note about it in the docs.

Reply:
I can add the same thing for GET also and yes it needs a little bit of change in code to add other ones.

Reply:
OK